herbert_core/utilities/misc/hlp_serialize.m

```diff
@@ -308,37 +308,37 @@
             % scoped function: Tag and Parentage
             m = [uint8(153); serialize_cell(rep.parentage)];
         otherwise
-            warn_once('hlp_serialize:unknown_handle_type','A function handle with unsupported type "%s" was encountered; using a placeholder instead.',rep.type); 
+            warn_once('hlp_serialize:unknown_handle_type','A function handle with unsupported type "%s" was encountered; using a placeholder instead.',rep.type);
             m = serialize_string(['<<hlp_serialize: function handle of type ' rep.type ' unsupported>>']);
     end
 end
 
 % *container* class to byte
 function b = class2tag(cls)
-	switch cls
-		case 'string'
-            b = uint8(0);
-		case 'double'
-			b = uint8(1);
-		case 'single'
-			b = uint8(2);
-		case 'int8'
-			b = uint8(3);
-		case 'uint8'
-			b = uint8(4);
-		case 'int16'
-			b = uint8(5);
-		case 'uint16'
-			b = uint8(6);
-		case 'int32'
-			b = uint8(7);
-		case 'uint32'
-			b = uint8(8);
-		case 'int64'
-			b = uint8(9);
-		case 'uint64'
-			b = uint8(10);
-              
+        switch cls
+                case 'string'
+                  b = uint8(0);
+                case 'double'
+                        b = uint8(1);
+                case 'single'
+                        b = uint8(2);
+                case 'int8'
+                        b = uint8(3);
+                case 'uint8'
+                        b = uint8(4);
+                case 'int16'
+                        b = uint8(5);
+                case 'uint16'
+                        b = uint8(6);
+                case 'int32'
+                        b = uint8(7);
+                case 'uint32'
+                        b = uint8(8);
+                case 'int64'
+                        b = uint8(9);
+                case 'uint64'
+                        b = uint8(10);
+
         % other tags are as follows:
```

Comment:
I believe because they are never assigned here, but only here for reference.

Reply:
If the other tags aren't assigned here, where are they assigned? It would be helpful to extend the comment to explain that.
An ideal implementation the type->tag and tag->type code would be defined in one place to make it easier to check consistency and completeness.

Reply:
This is how it is designed in the new system. I don't know why they decided that would be sensible.